sklearn/covariance/graph_lasso_.py

```diff
@@ -280,6 +280,10 @@ class GraphLasso(EmpiricalCovariance):
         The tolerance to declare convergence: if the dual gap goes below
         this value, iterations are stopped.
 
+    enet_tol : positive float, optional
+        The tolerance for the elastic net solver used to calculate the descent
+        direction. Only used for mode='cd'.
```

Comment:
Maybe you should add a sentence here to be even more explicit on how different this from the 'tol' parameter.